identity/client/e2e/tests/groups.spec.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import { expect } from "@playwright/test";
+import { test } from "../text-fixtures";
+import { Paths } from "../utils/paths";
+import { relativizePath } from "../utils/relativizePaths";
+import { LOGIN_CREDENTIALS } from "../utils/constants";
+import { waitForItemInList } from "../utils/waitForItemInList";
+
+import { IS_GROUPS_BASE_PAGE_INTEGRATED } from "../../src/feature-flags";
+
+const NEW_GROUP = {
+  groupId: "testgroupid",
+  name: "Test Group",
+  description: "test description",
+};
+
+const EDITED_GROUP = {
+  name: "Edited Group",
+  description: "edited description",
+};
+
+test.beforeEach(async ({ page, loginPage, groupsPage }) => {
+  await loginPage.navigateToLogin();
+  await loginPage.login(LOGIN_CREDENTIALS);
+  await expect(page).toHaveURL(relativizePath(Paths.users()));
+  await groupsPage.navigateToGroups();
+  await expect(page).toHaveURL(relativizePath(Paths.groups()));
+});
+
+test.describe[IS_GROUPS_BASE_PAGE_INTEGRATED ? "serial" : "skip"](
```

Comment:
I think we just do not need a feature flag for this since it is not really a feature. Can we just skip it directly instead.

Reply:
I don't see any downsides to having the feature flag, but I do see one benefit which is to concentrate the "TODOs" for unskipping all in the feature flags file (along with the link for the issue that unblocks it), then it becomes easy to find and anyone who will work on these integration/unskipping tasks already knows where to look first, and from there it's just finding where the feature flag is being used with a simple search. What do you think of this approach?

Reply:
Sure, I do not mind keeping them if you prefer that approach.
It just that we are adding those for a short term period, and they add extra logic that makes removing them later more difficult.